float64 pitch
float64 roll
float64 yaw
float64 acc_x
float64 acc_y
float64 acc_z
float64 gyro_x
float64 gyro_y
float64 gyro_z
float64 paro
float64 PH
float64 water
float64 temp
float64 mag1
float64 mag2
float64 mag3